examples/text_labels_and_annotations/mathtext_fontset_example.py

```diff
@@ -0,0 +1,82 @@
+"""
+========================
+Mathtext Fontset Example
+========================
+
+A simple example showcasing the new *mathtext_fontset* parameter that can 
+change the family of fonts for each individual text element in a plot.
+
+**OBS: The following code will only work on matplotlib version 3.4 and above.**
```

Comment:
Strike this...

Reply:
I think @jklymak only meant the version bit.